engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -780,14 +780,22 @@ public void monitorHeartbeat(boolean waitForRecovery)
           //LOG.debug("{} {} {}", c.getExternalId(), currentTms - sca.createdMillis, this.vars.heartbeatTimeoutMillis);
           // container allocated but process was either not launched or is not able to phone home
           if (currentTms - sca.createdMillis > 2 * this.vars.heartbeatTimeoutMillis) {
-            LOG.info("Container {}@{} startup timeout ({} ms).", c.getExternalId(), c.host, currentTms - sca.createdMillis);
+            LOG.error("Container {}@{} startup timeout ({} ms).", c.getExternalId(), c.host, currentTms - sca.createdMillis);
```

Comment:
@Hitesh-Scorpio @vrozov this is incorrect use of logging and I want to discuss this before it spreads further. That there can be problems with worker containers is an expected scenario in a distributed system and as we can see Apex handles the condition.  ERROR should be reserved for problems internal to Apex that cannot be handled by the system. Please undo this change.